plugins/do-kvm.py

```diff
@@ -64,11 +64,17 @@ def remove_mounts():
 
 
 def is_passable_mount(v):
+    print ("Checking mount: {}".format(v))
     if v[2] == "nfs" or v[2] == "nfs4":
+        return False
+    if v[1].startswith("/images/"):
+        print ("YES!!!")
+        return True
+    if v[1].startswith("/plugins"):
```

Comment:
HPCAC nodes are diskless. Here is how plugins are mounted:
 
```
Evaluating: /plugins
v is: ['tmpfs', '/plugins', 'tmpfs', 'ro,relatime,mode=555', '0', '0']
Passing: /plugins
```

Here is from a working system:
```
['/dev/sda5', '/plugins', 'ext3', 'ro,relatime', '0', '0']
```

Reply:
AFAIK, docker can't mount tmpfs, need to think about workaround

Reply:
It does work if we add the above

Reply:
So I think it should work as is